### DISCUSSION TITLE
Can't connect anymore - Successful - Waiting for image

### DISCUSSION THREAD

**Discussion by wiesl** [rustdesk/rustdesk#9124] [General]
### Bug Description

In an **existing working config** with the following setup:
- Windows 10 Clients, latest version
- Own Rustdesk Server under Linux with docker image (latest version, looks anyway outdated)

Suddenly I can't connect anymore:
- Successful - Waiting for image
- Then after around 20s I get: Connection Error - bytes remaining on stream
- repeats forever

`Started with version 1.2.6` on Windows 10 sides, but then the error occoured. Updated then to 1.2.7 with same behaviour. Also same behaviour on 1.3.0.

### How to Reproduce

Setup as above

### Expected Behavior

Get screen and work remotely

### Operating system(s) on local side and remote side

Windows 10 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.3.0 -> 1.3.0

### Screenshots

See explanation above

**Comment by rustdesk** [comment]
@21pages

**Comment by 21pages** [comment]
Could you use the program below?

https://github.com/21pages/rustdesk/releases/download/nightly/rustdesk-1.3.0-x86_64-waiting-for-image.exe

If installed on controlled side, please see whether "rustdesk --server" are restarted.

Log postion:

Control side:

Portable: `%AppData%\RustDesk\log\rustdesk_rCURRENT.log`
Installed:  `%AppData%\RustDesk\log\rustdesk_rCURRENT.log`

Controlled side:

Portable: `%AppData%\RustDesk\log\rustdesk_rCURRENT.log`
Installed: `C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\log\server\`

This is the log commit: https://github.com/21pages/rustdesk/commit/99f2a4ed8080c4ec9512951c9a00d36e80da5b6b

**Reply by wiesl** [reply]
Thnx for the prompt response. Will try it later today in the evening.

**Comment by cesar93600** [comment]
Hello, I had the same problem.
Version 1.3.3 (installed or not).
From 2 machines Windows (Server and Desktop) to a Windows 10 Client.
It was not a network problem, I could transfer files in the background.
I **disabled "Capture screen using DirectX"**.
Problem solved

**Reply by 21pages** [reply]
@cesar93600  Thank you for your feedback. I would like to investigate the issue with DirectX. Could you please assist me?

Run the program below on the controlled side and enable the `Capture screen using DirectX` option.
When you see `waiting for image`,  please upload the log file from the controlled side. 

https://github.com/21pages/rustdesk/releases/download/directx/rustdesk-1.3.5-x86_64.exe

Log file path:
* If not installed on the controlled side: `%AppData%\RustDesk\log\RustDesk_rCURRENT.log`
* If installed on the controlled side: `C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\log\server\RustDesk_rCURRENT.log`

The log file contains lots of `====DEBUG====` lines.  Remove personal information before upload the file.

**Comment by mrschwarz50** [comment]
I am having the same problem connecting to a Windows 11 machine from another Windows 11 machine. I am using a self-hosted server and am running version 1.4.0 on both machines. I have other machines that I can successfully connect to. It's just one machine.

I have uninstallled the client app. Deleted Rustdesk folders in the appdata folders in the user subdirectory. After restarting the client and reinstalling the app, the client still has the ID server address and key so that information must be stored somewhere else.

Don't know if the instructions above still apply or if there is something else to do with the newer version of Windows and Rustdesk.

Thanks.

**Reply by open-trade** [reply]
How about try different codec (on controlling side) https://github.com/rustdesk/rustdesk/wiki/FAQ#how-to-change-codec?

**Reply by open-trade** [reply]
and how about disable "Capture screen using DirectX" (on controlled side). https://github.com/rustdesk/rustdesk/discussions/9124#discussioncomment-11509296

**Reply by mrschwarz50** [reply]
Thanks for the response. I tried all the different codecs without any improvement. I also disabled "Capture screen using DirectX". That didn't work either.

**Reply by MortalM** [reply]
yes i have same  question 
<img width="1788" height="1002" alt="image" src="https://github.com/user-attachments/assets/cc2eaf00-a025-47fe-8399-4f88d3d2f65d" />
